src/components/Card/index.tsx

```diff
@@ -0,0 +1,45 @@
+import Image from 'next/image';
+import React from 'react';
+
+type Props ={
+  time: number;
+  content: string;
+  userIcon: string;
+  userName: string;
+  HeyCount: number;
+};
+
+export const Card = (props:Props) => (
```

Comment:
props の受け取りは分割代入での統一でお願いします :pray:

```suggestion
export const Card = ({ time, content, userIcon, userName, heyCount }:Props) => (
```

Reply:
分割代入に変更いたしました🙏✨